sample-apps/donet-sample-app/Startup.cs

```diff
@@ -0,0 +1,63 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using OpenTelemetry;
+using OpenTelemetry.Contrib.Extensions.AWSXRay.Trace;
+using OpenTelemetry.Resources;
+using OpenTelemetry.Trace;
+using System;
+
+namespace integration_test_app
```

Comment:
I don't think this naming is appropriate. Yes this application will have integration tests performed against it but it's purpose is to be a sample app for instrumenting .NET applications.